roles/nodered/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+nodered_install: True
+nodered_enabled: False
+nodered_user: admin
```

Comment:
Please change 'admin' to 'Admin' to be consistent with other Internet-in-a-Box playbooks, Thanks!